Dockerfile

```diff
@@ -1,14 +1,22 @@
 FROM python:3.7.1-stretch
 
-ENV PYTHONUNBUFFERED=1
-
-RUN apt-get update && apt-get install -y vim tmux
+ENV PYTHONUNBUFFERED    1 
```

Comment:
Since you started working on this, can you add a comment so as to why we need each variable?

Reply:
so the two paths are needed by `charm`. I think `PYTHONBUFFERED` has got to do something with the buffering of stream when spitting data on to the console.